core/acl.py

```diff
@@ -113,3 +119,34 @@ async def repeat(self, ctx, *, input: str):
 
     # user's roles are not mapped to any ACL group, return default
     return rule.default
+
+
+def version(major: int, minor: int, micro: Optional[int] = None):
+    """Decorator that specifies minimum discord.py version"""
+
+    def decorator(func):
+        @wraps(func)
+        async def wrapper(*args, **kwargs):
+            version_info = discord.version_info
+            version_check = True
+
+            if version_info.major == major:
+                if version_info.minor == minor:
+                    if micro is None or version_info.micro >= micro:
+                        version_check = False
+                elif version_info.minor > minor:
+                    version_check = False
+            elif version_info.major > major:
+                version_check = False
+
+            if not version_check:
+                return await func(*args, **kwargs)
+
+            if isinstance(args[1], Context):
+                await args[1].message.reply(tr("version", "reply"))
```

Comment:
Now that I think about it sometimes we may want to return plain `False` with no message. Can you add named argument to the check? `@check.version(2, 0, reply=False)`/`@check.version(2, 0, reply=True)`.